samples/rpc-priority-transaction.js

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// sample-metadata:
+//  title: Read and write transaction with RPC priority
+//  usage: node rpc-priority-transaction.js <INSTANCE_ID> <DATABASE_ID> <PROJECT_ID>
+
+'use strict';
+
+async function main(instanceId, databaseId, projectId) {
+  // [START spanner_rpc_priority_transaction]
+  // Imports the Google Cloud client library.
+  const {Spanner, protos} = require('@google-cloud/spanner');
+  const Priority = protos.google.spanner.v1.RequestOptions.Priority;
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
+  function transactionWithRpcPriority(instanceId, databaseId) {
+    // Gets a reference to a Cloud Spanner instance and database.
+    const instance = spanner.instance(instanceId);
+    const database = instance.database(databaseId);
+
+    database.runTransaction(
+      {
```

Comment:
I think we should add a comment to explain what this does. At least, I don't know what the effect of this will be. Does it mean that all RPCs in this transaction will have priority `LOW`? Or does it only mean that the `Commit` RPC will be invoked with priority `LOW`?

Reply:
You are right, it does not make any sense to add priority in run transaction method. Got it confirmed from Thiago as well that passing a priority to a transaction applies to the commit call. Made the changes as required.